package.json

```diff
@@ -30,7 +30,7 @@
     "mustache": "2.3.0"
   },
   "devDependencies": {
-    "truffle": "4.0.1",
+    "truffle": "v4.1.8",
```

Comment:
why v4.1.8?

Reply:
Truffle 4.0.1 internally uses solc version 0.4.18. Which uses the old syntax of solidity. Upgrading truffle version to v4.1.8 also upgrades newer syntax of solidity(0.4.23).